lib/compat/wordpress-6.8/class-gutenberg-rest-post-types-controller-6-8.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Post_Types_Controller_6_8 class
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Gutenberg_REST_Post_Types_Controller_6_8 class
+ *
+ * Add Block Editor default rendering mode to the post type response
+ * to allow enabling/disabling at the post type level.
+ */
+class Gutenberg_REST_Post_Types_Controller_6_8 extends WP_REST_Post_Types_Controller {
+	/**
+	 * Add Block Editor default rendering mode setting to the response.
+	 *
+	 * @param  WP_Post_Type    $item    Post type object.
+	 * @param  WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $item, $request ) {
```

Comment:
Not sure if this was discussed before, or if there is some hidden caveat, but to merely add a new field to an existing endpoint, you don't need to override the entire controller. There is an API for that:
```
register_rest_field( 'type', 'default_rendering_mode', ... );
```

Reply:
Update: now I see that this was already solved by @Mamaduka in #68549 and #67867 🚀 Nice coincidence.

Reply:
Thanks, @jsnajdr!